assets/app/view/game/corporation.rb

```diff
@@ -189,7 +189,7 @@ def render_tokens
           }
           img_props[:style][:filter] = 'contrast(50%) grayscale(100%)' if token.used
 
-          token_text = i.zero? ? @corporation.coordinates : token.price
+          token_text = i.zero? ? @corporation.coordinates : token.hex_name
```

Comment:
you can just do token.city.hex.name instead

Reply:
if you'd prefer, you could add a 

def hex to the token and call token.hex.name, but up to you